includes/class-wc-stripe-api.php

```diff
@@ -173,21 +190,42 @@ public static function get_idempotency_key( $api, $method, $request ) {
 	 * @throws WC_Stripe_Exception
 	 */
 	public static function request( $request, $api = 'charges', $method = 'POST', $with_headers = false ) {
-		WC_Stripe_Logger::log( "{$api} request: " . print_r( $request, true ) );
-
 		$headers = self::get_headers();
 
 		$idempotency_key = apply_filters( 'wc_stripe_idempotency_key', self::get_idempotency_key( $api, $method, $request ), $request );
 		if ( $idempotency_key ) {
 			$headers['Idempotency-Key'] = $idempotency_key;
 		}
 
+		$request = apply_filters_deprecated(
+			'woocommerce_stripe_request',
```

Comment:
Shouldn't this be `woocommerce_stripe_request_body` instead (based on the original code)?